app/src/test/java/io/github/jbellis/brokk/tools/SkeletonPrinter.java

```diff
@@ -234,14 +249,16 @@ private static void printDirectorySkeletons(Path directory, Language language) {
                     continue;
                 }
 
+                var projectFile = singleFileProject.getAllFiles().iterator().next();
```

Comment:
Is this always guaranteed to be non-empty? Just wondering if we should add a check here or if it's related to the outer loop in some way.